integration-tests/suites/udp.go

```diff
@@ -0,0 +1,96 @@
+package suites
+
+import (
+	"time"
+
+	"github.com/stackrox/collector/integration-tests/suites/common"
+	"github.com/stretchr/testify/assert"
+)
+
+type UdpTestSuite struct {
```

Comment:
Can we have some docs here describing what this test is for and how it works

Reply:
Done